samples/net/openthread/coap_client/README.rst

```diff
@@ -24,34 +25,48 @@ User interface
 
 The following board buttons are used by this sample:
 
-BUTTON 1:
-   Pressing results in sending a unicast LIGHT_TOGGLE message to the /light resource on a paired device. If no device is paired with the specific client node, pressing the button has no effect.
+Button 1:
+  Pressing results in sending a unicast LIGHT_TOGGLE message to the /light resource on a paired device. If no device is paired with the specific client node, pressing the button has no effect.
 
-BUTTON 2:
-    Pressing results in sending a multicast LIGHT_ON or a LIGHT_OFF message (alternatively) to the /light resource. Sending a LIGHT_ON/LIGHT_OFF message instead of LIGHT_TOGGLE allows to synchronize the state of the LEDs on several server nodes.
+Button 2:
+  Pressing results in sending a multicast LIGHT_ON or a LIGHT_OFF message (alternatively) to the /light resource. Sending a LIGHT_ON/LIGHT_OFF message instead of LIGHT_TOGGLE allows to synchronize the state of the LEDs on several server nodes.
 
-BUTTON 3:
-    Pressing results in sending a multicast pairing request to the /provisioning resource.
+Button 3:
+  Pressing results in sending a multicast pairing request to the /provisioning resource.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/net/openthread/coap_client`
+
+.. include:: /includes/build_and_run.txt
 
 
 Testing
 =======
 
-After programming the sample to your board, test it by performing the following steps:
+After building the sample, test it by performing the following steps:
+
+#. Program one or more boards with the Thread Simple CoAP Client and the Thread Simple CoAP Server.
+#. Turn on the Simple CoAP Server or Simple CoAP Client node and wait for a while.
+#. Turn on any of the other nodes. They will enter the network as Children, and Simple CoAP Client or Simple CoAP Server nodes will gradually become Routers.
```

Comment:
OK, so you need even more nodes? Please add this information to Requirements for this sample. But I think we should stick here to the simplest possible configuration, so if the sample can work with one or two server nodes, as suggested by the description of Button 2, we should stick to this minimal scenario.